src/analysis/ConfigurationExt.cpp

```diff
@@ -91,6 +109,58 @@ namespace espressopp {
       //end = coordinates.end();
     }
 
+
+    size_t ConfigurationExtIterator::Id()
+    {
+      if (it == end) {
+        PyErr_SetString(PyExc_StopIteration, "No more data.");
+        boost::python::throw_error_already_set();
+      }
+
+      size_t id = (*it).first;
+      return id;
+    }
+
+
+    //bad performance - or the compiler is smart
+    const RealND ConfigurationExtIterator::Properties()
+    {
+      if (it == end) {
+        PyErr_SetString(PyExc_StopIteration, "No more data.");
+        boost::python::throw_error_already_set();
+      }
+
+      RealND props = (*it).second;
+      return props;
+    }
+
+
+    /*
+        Real3D ConfigurationExtIterator::nextCoordinates()
```

Comment:
I guess you're referring to Real3D ConfigurationExtIterator::nextCoordinates(), but I see const RealND ConfigurationExtIterator::Properties() highlighted... 
Yes, Real3D ConfigurationExtIterator::nextCoordinates() is not used and probably the previous developers commented out thus just using const RealND ConfigurationExtIterator::Properties() for both coordinates and velocities.
Just kept as it was. We can think about removing this commented code...
Waiting feedback.